### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by albertchen-okta**
## Description:
- **What's changed?** 
- this UI was updated the the Branding GA last Nov 2021, so the instructions have been incorrect since then
-
- **Is this PR related to a Monolith release?**
- none

### Resolves:

* none

### CODE REVIEW

**Review by brentschaus-okta — APPROVED** [review]
LGTM

**Review by francoislehoux-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 1]
```diff
@@ -47,12 +47,10 @@ The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, a
...
+The HTML editor that is provided on the **Error pages code editor** section of **Customizations** allows you to modify any HTML, CSS, and JavaScript on the error page.
 
-1. In the Admin Console, select **Settings** and then **Customizations**.
-1. Click the **Custom Error Pages** tab.
-> **Note**: In Okta Identity Engine, select **Customizations** and then **Error pages code editor** to find the error pages editor.
-1. Make changes directly in the embedded **Custom Error Pages** editor.
+1. In the Admin Console, select **Customizations** and then **Error pages code editor**.
+1. Make changes directly in the embedded **Error Pages** editor.
 1. Click **Preview** to preview your changes before you publish.
 1. Click **Reset to Default** if you need to remove all of your customizations and restore the default HTML/CSS and JavaScript code.
```
Comment (gaurigattani-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:55]: We no longer have a 'reset to default'. 

if you need to remove all of your customizations and restore the default - user needs to toggle to variant view. 

We do have a 'revert changes' to go back to published template. 

Do we want to include these?

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 2]
```diff
@@ -47,12 +47,10 @@ The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, a
...
 1. Click **Save and Publish** to commit your changes.
```
Comment (gaurigattani-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:56]: This is now publish only. 

Also, briefly we can describe auto save?

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md (hunk 1 above)
Comment (albertchen-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:55]: Keep in mind this PR is for the current state (pre-EA for code editors). I will have a new PR ready for when we deploy 2022.04.3 to preview

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md (hunk 2 above)
Comment (albertchen-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:56]: same comment as above

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md (hunk 1 above)
Comment (gaurigattani-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:55]: ok, my bad.

**Pull request merged**